src/SolidGui/Program.cs

```diff
@@ -95,7 +95,6 @@ private static void SetupErrorHandling()
         {
             ExceptionHandler.Init(new WinFormsExceptionHandler());
             Logger.Init();
-            ErrorReport.Init("solid@projects.palaso.org");

```

Comment:
I believe you also need to remove the nearby dropdown containing the "email" option.

Reply:
That sounds like a great idea, however the windows forms dialog is part of SIL.Windows.Forms in libpalaso and, having a look at https://github.com/sillsdev/libpalaso/blob/e5745ec04c507a18bfe6b67c29c177713ec1a15d/SIL.Windows.Forms/Reporting/ExceptionReportingDialog.cs#L284 the private method setupMethodCombo() where the combo box is populated is not customizable or exposed.  So that would require more coordination to release a new version of libpalaso with changes that accommodate what we'd like.  So basically not worth the effort at this point.

Thanks for thinking about that though.